javaslang/src/test/java/javaslang/collection/IntMap.java

```diff
@@ -292,6 +292,11 @@ public long estimateSize() {
             public int characteristics() {
                 return spliterator.characteristics();
             }
+
+            @Override
+            public Comparator<? super T> getComparator() {
+                return null;
+            }
```

Comment:
The super impl Spliterator.getComparator() throw an IllegalStateException by default.

Is it really necessary to return null? If null is used somewhere it will throw a NPE, which is roughly the same as throwing an IllegalStateException.

I'm just curious - I'm sure there is a reason!